code/datums/weather/weather_types/acid_rain.dm

```diff
@@ -21,7 +21,7 @@
 	protect_indoors = TRUE
 	target_trait = ZTRAIT_ACIDRAIN
 
-	immunity_type = ACID // temp
+	immunity_type = "acid" // temp
```

Comment:
why are you turning defines into strings what

Reply:
Consistency. Ash storms, snow storms, and void storms do not use defines.

Reply:
Hmm, it'd probably be better in the long term to swap these for defines instead, since with defines you could just jump to where all the weather defines are to look at all the different immunity_type options.

Reply:
what the others said, if you find that some strings are their define and some aren't, it's better to turn the ones that aren't into defines instead of the other way around

Reply:
> Consistency. Ash storms, snow storms, and void storms do not use defines.

You should be adding consistency by making code better, not more shit